VimeoNetworking/Sources/VimeoClient.swift

```diff
@@ -348,37 +345,42 @@ final public class VimeoClient
         
         do
         {
-            let modelObject: ModelType = try VIMObjectMapper.mapObject(responseDictionary, modelKeyPath: request.modelKeyPath)
+            let modelObject: ModelType = try VIMObjectMapper.mapObject(responseDictionary: responseDictionary, modelKeyPath: request.modelKeyPath)
             
             var response: Response<ModelType>
             
-            if let pagingDictionary = responseDictionary[self.dynamicType.PagingKey] as? ResponseDictionary
+            if let pagingDictionary = responseDictionary[Constants.PagingKey] as? ResponseDictionary
             {
-                let totalCount = responseDictionary[self.dynamicType.TotalKey]?.longValue
-                let currentPage = responseDictionary[self.dynamicType.PageKey]?.longValue
-                let itemsPerPage = responseDictionary[self.dynamicType.PerPageKey]?.longValue
+                
+                guard let totalCount = responseDictionary[Constants.TotalKey] as? NSNumber,
+                    let currentPage = responseDictionary[Constants.PagingKey] as? NSNumber,
+                    let itemsPerPage = responseDictionary[Constants.PerPageKey] as? NSNumber else
```

Comment:
`responseDictionary` should be `[AnyHashable: Any]`, but we're passing these values into an initializer for `Response` that requires `Int`s and `String`s. So we can just cast directly to `Int` here instead of going to NSNumber first.